### PR TITLE
Fix needed settings to enable beaker

### PR DESCRIPTION
**Pull request by TheMeier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,25 +220,28 @@ following entry to your `.sync.yml` and run `pdk update`.
...
+      - gem: beaker-abs
+        from_env: BEAKER_ABS_VERSION
+        version: '~> 0.1'
+      - gem: beaker-pe
+      - gem: beaker-hostgenerator
+        from_env: BEAKER_HOSTGENERATOR_VERSION
+      - gem: beaker-rspec
+        from_env: BEAKER_RSPEC_VERSION
+.travis.yml:
+  bundler_args: --with system_tests
```
Comment (scotje) [inline README.md]: We should probably add the equivalent settings for Appveyor and Gitlab-CI configs.

Comment (TheMeier) [reply README.md]: @scotje like that?

Comment (scotje) [reply README.md]: Yep! It looks like this has uncovered a bug in the gitlab customization though, I'm still working through that. Also it looks like the config for `.gitlab-ci.yml` has a `beaker` key that we might also want to suggest setting?

Comment (TheMeier) [reply README.md]: @scotje added `beaker: true` I wonder where one could fine a more complete example showing the use of `bekaer.tags` and `beaker.variables`

**Comment by scotje** [comment]
After #244 is merged we can merge this, although I think the `.gitlab-ci.yml` config needs to include the flags from `config_defaults`, I will add a suggestion to the code.

File: README.md [hunk 2]
```diff
@@ -221,25 +221,33 @@ following entry to your `.sync.yml` and run `pdk update`.
...
+.gitlab-ci.yml:
+  bundler_args: --with system_tests
```
Comment (scotje) [inline README.md]:
```suggestion
  bundler_args: --with system_tests --path vendor/bundle --jobs $(nproc)
```

**Comment by scotje** [comment]
Thanks for your contribution @TheMeier !

**Pull request merged**